static/js/graphs_page.js

```diff
@@ -584,7 +594,7 @@ var graphPage = {
         graphPage.cyGraph.layout(layout);
 
     },
-    saveLayout: function (layoutName) {
+    saveLayout: function (layoutName, modalNameId) {
```

Comment:
I dont like the change to `saveLayout` method. It doesn't make sense in terms of the method  signature. Here are few questions for you -

- Can we rename modalNameId to a more intuitive name?
- Can move modalNameID out saveLayout method? It seems that we need it only for closing the modal.